x-pack/plugins/security_solution/public/overview/components/overview_cti_links/index.tsx

```diff
@@ -33,4 +32,10 @@ const ThreatIntelLinkPanelComponent: React.FC<ThreatIntelLinkPanelProps> = (prop
 
 ThreatIntelLinkPanelComponent.displayName = 'ThreatIntelDashboardLinksComponent';
 
-export const ThreatIntelLinkPanel = React.memo(ThreatIntelLinkPanelComponent);
+export const ThreatIntelLinkPanel = React.memo(
+  ThreatIntelLinkPanelComponent,
+  (prevProps, nextProps) =>
```

Comment:
Is this function implicitly _excluding_ some prop from the comparison? If it is, a comment to that effect would be good for posterity. If it's not, then we can probably do away with this function.

Reply:
removed